app/feedstock/[slug]/stores.tsx

```diff
@@ -0,0 +1,166 @@
+                sx={{
+                  color: color,
+                  fontSize: [2, 2, 2, 3],
+                  textTransform: 'uppercase',
+                }}
+                onClick={() => setExpanded((prev) => !prev)}
+              >
+                <Text sx={{ mr: [2] }}>{store.name || store.id}</Text>
+                <Expander value={expanded} />
+              </Button>
+            </TooltipWrapper>
+          </Box>
+        </Column>
+      </Row>
+
+      <AnimateHeight
+        duration={100}
+        height={expanded ? 'auto' : 0}
+        easing={'linear'}
+      >
+        <Row sx={{ mt: 4 }}>
+          <Column start={[1, 1, 2, 2]} width={[6, 4, 3, 3]}>
+            <Box sx={{ mt: 2 }}>
+              <Box sx={{ mb: 2 }}>
+                {' '}
+                <Text
+                  // @ts-ignore
+                  sx={labelStyle}
+                >
+                  Data Viewer
+                </Text>
+              </Box>
+
+              <DataViewer store={store} color={color} />
+            </Box>
+          </Column>
+          <Column start={[1, 5, 7, 7]} width={[6, 4, 5, 5]}>
+            <Box sx={{ mt: 2 }}>
+              <Box sx={{ mb: 2 }}>
+                <Text
+                  // @ts-ignore
+                  sx={labelStyle}
+                >
+                  Load Dataset
+                </Text>
+              </Box>
+              <CodeSnippet url={store.url} color={color} />
+            </Box>
+          </Column>
+
+          <Column start={[1, 5, 7, 7]} width={[6, 4, 5, 5]}>
+            <Box sx={{ mt: 2 }}>
+              <TooltipWrapper
+                tooltip={tooltipContent}
+                color={color}
+                expanded={!store.public && tooltipExpanded}
+                setExpanded={setTooltipExpanded}
+              >
+                <Box sx={{ mb: 2 }}>
+                  <Text
+                    // @ts-ignore
+                    sx={labelStyle}
+                  >
+                    Metadata
+                  </Text>
+                </Box>
+              </TooltipWrapper>
+
+              <Suspense fallback={<div>Loading dataset representation...</div>}>
+                <DatasetReprFetcher
+                  store={store}
+                  serverDataPromise={serverDataPromise}
+                />
+              </Suspense>
+            </Box>
+          </Column>
+        </Row>
+      </AnimateHeight>
+    </Box>
+  )
+}
+
+export const FeedstockStores: React.FC<{
+  stores: Store[] | undefined
+  color: string
+}> = ({ stores, color }) => {
+  return (
+    <>
+      <Row>
+        <Column start={[1, 2, 2, 2]} width={[6]}>
+          <Box sx={{ mb: 4, mt: 2 }}>
+            <Text
+              sx={{
+                color: 'muted',
+                fontSize: 0,
+                fontFamily: 'mono',
+                letterSpacing: 'mono',
+                textTransform: 'uppercase',
+              }}
+            >
+              Available Stores
```

Comment:
```suggestion
              Access Data
```
I think that 'stores' might not be familiar to some people.